data/companies.yml

```diff
@@ -207,6 +207,10 @@ integrations:
     description: "Application layer protection for Istio service mesh"
     details:
       - "Provides complete layer 7 protection for microservices architecture applications."
+  - name: "Solo.io"
+    logo: "/logos/solo.png"
+    url: "https://www.solo.io/"
+    description: "Solo.io offers Gloo Mesh, building on Istio, with expert support."
```

Comment:
Solo and Gloo Mesh are already referred to in the "providers" section of the site.

The integrations section is intended for software, not companies; you could list something like Squash in this space, but the bar will be "specific Istio features or integrations".

Reply:
@craigbox Would our work on [WebAssembly](https://github.com/solo-io/wasm) qualify since it can be used with Envoy Proxy and Istio freely and independently of our paid Gloo products?

Reply:
the intention is for it to be for products or services, so I suspect https://webassemblyhub.io/ would qualify